packages/vats/src/localchain.js

```diff
@@ -118,7 +118,9 @@ export const prepareLocalChainAccountKit = (zone, { watch }) =>
          */
         onFulfilled(brand, { payment, optAmountShape }) {
           const purse = E(this.state.bank).getPurse(brand);
-          return E(purse).deposit(payment, optAmountShape);
+          return E(purse)
+            .deposit(payment, optAmountShape)
+            .then(() => {});
```

Comment:
This doesn't seem to be the correct place for this - we should only care about the return signature of `deposit` in `local-orchestration-account.js`. We currently use the `returnVoidWatcher` (handler) there, so I believe this commit can be removed

Reply:
Thanks for the catch. I revised to fix the type annotation instead